templates/webapi/test/overview.html.ep

```diff
@@ -20,6 +21,16 @@
             % if ($build) {
                 build <%= $build %>
             % }
+            <div class="time-params">
+                % if ($until) {
+                    from <abbr class="timeago" title="<%= $until %>Z"><%= $until %>Z</abbr>,
+                    <a href="<%= url_with->query({t => undef}) %>">show latest jobs</a>
+                % }
+                % else {
+                    showing latest jobs,
+                    <a href="<%= url_with->query({t => format_time(DateTime->now)}) %>">overview fixed to the current time</a>
```

Comment:
As a user I'd wonder what that link would be useful for. From the user story in the ticket I guess this only makes sense for builds far in the past.

Reply:
The link is actually used to have a reference of the overview as it looks at this point to be able to view it again in the future. That might be useful for sharing the link or putting it as reference into a ticket. I suppose that was actually the initial use-case from which the ticket has been derived. That's why @okurz suggested to add such a link in his previous comment and I've added it. Of course the feature could be made more generic (e.g. by adding a datetime picker) but I guess it is good for now. (People can of course already add an arbitrary timestamp manually.)

Reply:
Exactly. The idea is: what is the current present becomes "far in the past" if you wait long enough, like in every second bug when someone looks at it after half a year ;)

Reply:
How about rendering the date/time as the text for the link .e.g `Showing jobs from 2020-10-23 18:18` like we do elsewhere, and like is done on GitHub, social media, etc (using whatever format we use, this is just an example)

Reply:
Please focus on the most minimal functionality for now. As I described in https://github.com/os-autoinst/openQA/pull/3482#pullrequestreview-515088379 "For now I would not care about a datepicker or anything complicated like that. We can still think about this in case people ask for it and the demand is there."